guides/accounting-guide.rst

```diff
@@ -11,7 +11,7 @@ Flux Accounting Guide
     documented in this guide may change with regularity.
 
     This document is in DRAFT form and currently applies to flux-accounting
-    version 0.18.1.
+    version 0.30.0.
```

Comment:
Needs to be bumped again.

Suggestion for later: move this document to the flux accounting project like we did with the admin guide and flux core.   and stop mentioning the version in the guide, since it's then more obvious that the guide applies to the current version.

(Side benefit: if you set up readthedocs infrastructure in flux-accounting then man pages for the commands would be easy to add)

Reply:
This is a good suggestion, thanks! I'll open an issue to move this document to flux-accounting 👍